command/crypto/otp/verify.go

```diff
@@ -21,11 +23,15 @@ func verifyCommand() cli.Command {
 		Name:   "verify",
 		Action: cli.ActionFunc(verifyAction),
 		Usage:  "verify a one-time password",
-		UsageText: `**step crypto otp verify** <secret-file>
+		UsageText: `**step crypto otp verify** **--secret**=<path>
 [**--period**=<seconds>] [**--skew**=<num>] [**--length**=<size>]
-[**--alg**=<alg>] [*--time**=<time|duration>]`,
+[**--alg**=<alg>] [*--time**=<time|duration>] [**-url**]`,
```

Comment:
I don't see the `--url` flag defined, and it has just one `-`.

Reply:
gofmt?

Reply:
Not sure what you mean by `gofmt`? `make lint` wasn't showing any errors.

Reply:
Same as before, if `gofmt` doesn't change it, then it should be ok.